tests/testthat/test-crunch-data-frame.R

```diff
@@ -17,37 +17,37 @@ with_mock_crunch({
         expect_equal(ds_df2$gender, gndr[new_order])
         expect_equal(nrow(ds_df2), 4)
     })
-    
+
     test_that("Extract methods work with CrunchDataFrames like they do with data.frames", {
         ds_new <- loadDataset("test ds")
-        # remove array variables because they are not equivalent in crdf and 
+        # remove array variables because they are not equivalent in crdf and
         # data.frames, since in the data.frames they have been flattened.
         ds_new <- ds_new[!{names(ds_new) %in% c("mymrset", "catarray")}]
-        
-        ds_df <- as.data.frame(ds_new)
-        true_df <- as.data.frame(ds_new, force = TRUE)
 
+        ds_df <- as.data.frame(ds_new)
+        true_df <- readRDS("dataset-fixtures/test_ds.rds")
+        true_df <- true_df[, c("birthyr", "gender", "location", "textVar", "starttime")]
         # single column/row extraction
-        expect_equivalent(ds_df[1,], true_df[1,])
+        all.equal(ds_df[1,], true_df[1,])
```

Comment:
What's the reason for the change from `expect_equivalent` to `all.equal`? If it's false, it doesn't look like this would throw an error in test_that

Reply:
Hmmm, I'm really not sure honestly. I'll revert this one=